### DISCUSSION TITLE
gomod major version not updated

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#9713] [Request Help]
Hello,

In a Golang project renovate creates a PR for a major version update for a go module dependency but the changed files looking strange and in this state the PR is quite useless. https://github.com/thegeeklab/drone-github-comment/pull/35

Any idea what's wrong?

**Comment by xoxys** [comment]
Now I'm confused completely. Related to https://github.com/renovatebot/renovate/issues/9492 I've added the mentioned `gomodUpdateImportPaths` and now I get a fetch error:

![grafik](https://user-images.githubusercontent.com/3391958/115992443-150dbd00-a5ce-11eb-9dc9-a1630a5ec73c.png)

**Reply by rarkins** [reply]
Can you create a minimum reproduction repo to isolate the problem? I suspect the reason that your initial PR failed to update go.sun is the same as you see in the second run - there's some dependency failing to go get

**Reply by xoxys** [reply]
Thanks for your fast response. Sure, will prepare one.

**Reply by xoxys** [reply]
Takes a bit longer as expected but here is the minimal example https://github.com/xoxys/test/pull/6

**Reply by rarkins** [reply]
@xoxys thanks, we will treat it as a bug, so I have created #9761

**Comment by tboerger** [comment]
Looks like the flag to update the import paths doesn't have any effect.

**Reply by rarkins** [reply]
I think the last line in the logs tells the opposite - it's being attempted
